definitions/firefox_ios.toml

```diff
@@ -441,7 +441,7 @@ client_id_column = "client_id"
 submission_date_column = "submission_date"
 
 [data_sources.new_profile_activation]
-from_expression = "`moz-fx-data-shared-prod.firefox_ios.new_profile_activation`"
```

Comment:
Alternatively, we could use `firefox_ios.firefox_ios_clients` here instead as the views also contains this field:
https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/firefox_ios/firefox_ios_clients/view.sql#L21